pose_estimation.orogen

```diff
@@ -332,3 +332,21 @@ task_context "OrientationInitialization" do
 
     periodic 0.01
 end
+
+task_context "UWPoseEstimatorFixedCovariance" do
+    subclasses "UWPoseEstimator"
+    needs_configuration
+
+    # Depth covariance element
+    property "cov_depth", "double"
+    # Orientation covariance matrix
+    property "cov_orientation", "/base/Matrix3d"
+end
+
+task_context "HighDelayPoseEstimatorFixedCovariance" do
+    subclasses "HighDelayPoseEstimator"
+    needs_configuration
+
+    # XY samples covariance matrix
+    property "cov_xy_samples", "/base/Matrix2d"
+end
```

Comment:
I think best you implement this in a package specific to tidewise. It is very specific to our current setup.

Moreover, the `subclasses` statement as you did is deprecated (there  is a big fat warning about it). You need to do `task_context "HighDelayPoseEstimatorFixedCovariance", subclasses: "pose_estimator::HighDelayPoseEstimator"`

Reply:
Figured we would use the fork, at what package do you think it would be better?

Reply:
As I mentioned in chat, I dont think it would be cleaner to use another package, since RBSFilter (the task both of these subclass from) uses `pose_estimationTypes.hpp`. Waiting on your opinion to proceed with the development of this.